test/groovy/CommonStepsTest.groovy

```diff
@@ -53,7 +53,13 @@ public class CommonStepsTest extends BasePiperTest{
             'piperExecuteBin',
             'prepareDefaultValues',
             'setupCommonPipelineEnvironment',
-            'buildSetResult'
+            'buildSetResult',
+            'malwareExecuteScan', // go pattern, script not needed
+            'mavenExecuteStaticCodeChecks',
+            'cloudFoundryCreateServiceKey',
+            'gctsCreateRepository',
+            'abapEnvironmentPullGitRepo',
+            'cloudFoundryDeleteService',
```

Comment:
This should not be necessary now as the test is now working for these steps.

Reply:
Ok, good to know. I think I also added this incorrectly in another PR. Thanks for the hint!